man/systemd.exec.xml

```diff
@@ -2912,7 +2912,12 @@ StandardInputData=SWNrIHNpdHplIGRhIHVuJyBlc3NlIEtsb3BzLAp1ZmYgZWVtYWwga2xvcHAncy
         <citerefentry><refentrytitle>journalctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>
         output, unless the <option>--namespace=</option> option is used.</para>
 
-        <xi:include href="system-only.xml" xpointer="singular"/></listitem>
+        <xi:include href="system-or-user-ns.xml" xpointer="singular"/>
+
+        <para>For services running in per-user instances of the service manager, note that it is also
+        required to manually start a system <literal>systemd-journald@<replaceable>NS</replaceable>.service</literal>
+        unit, as it is not possible to have an implicit dependency from a user unit to a system unit.
+        </para></listitem>
```

Comment:
hmm, we could of course look into making journald work in user mode? i think it mostly should work already?

Reply:
yes I already added an item for tomorrow's meeting agenda on this